app/controllers/ems_common.rb

```diff
@@ -561,6 +563,10 @@ def button
     end
   end
 
+  def provider_documentation_url
+    "http://manageiq.org/documentation/getting-started/#adding-a-provider"
```

Comment:
I would have thought that the `url` would be maintained by each EMS model.  Not sure I understand what having it hard-coded here does.